### PR TITLE
Messageobject support in `WitsmlExplorer.Frontend`

### LINKED ISSUE #163: User story:  MESSAGE Object - Frontend View
As a Data Manager, I want to be able to browse and view the MESSAGE object in WE

### Acceptance criteria ###
- [ ] The WE  frontend is extended to support browsing the MESSAGE object

Also, here are a few points that need to be addressed:
- [ ] Constraint 1
- [ ] Constraint 2

### Resources ###
-  Resource A
-  Resource B

### PR DESCRIPTION
**Pull request by steinsiv**
## Description
Fixes #163  Messageobjects.Frontend View

## Type of change
Objects for `Frontend` support of messageobject

* New feature (non-breaking change which adds functionality)
* Enhancement of existing functionality

## Impacted Areas in Application
`/WitsmlExplorer.Frontend`

## Checklist:

*Communication*
* [x] PR is related to an issue
* [ ] I have made corresponding changes to the documentation

*Code quality*
* [x] Code follows the style guidelines
* [x] I have self-reviewed my code
* [x] No new warnings are generated

*Test coverage*
* [x] Existing tests pass
* [x] New code is covered by passing tests

## Further comments
This PR is to support message objects in `Frontend` package and assumes support in `Witsml` and `WitsmlExplorer.Api` package.

### CODE REVIEW

**Review by kdi-chandra — APPROVED** [review]
Looks good

File: Src/WitsmlExplorer.Frontend/models/jobs/messageObjectReference.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export default interface MessageObjectReference {
+  wellUid: string;
+  wellboreUid: string;
```
Comment (kdi-chandra) [inline Src/WitsmlExplorer.Frontend/models/jobs/messageObjectReference.ts:4]: Should include messageUid as part of MessageObjectReference

**Review by kdi-chandra — APPROVED** [review]
Looks good with small comment. Tested locally, working as expected
- [x] Read issue and PR
- [x] Pulled branch and manually tested
- [x] Verify that PR resolves issue
- [x] Reviewed the code

File: Src/WitsmlExplorer.Frontend/models/jobs/messageObjectReference.ts (hunk 1 above)
Comment (steinsiv) [reply Src/WitsmlExplorer.Frontend/models/jobs/messageObjectReference.ts:4]: Thanks! 😃  Fixed

**Pull request merged**